charts/fluent-operator/templates/fluentbit-fluentBit.yaml

```diff
@@ -16,8 +16,8 @@ spec:
   {{- else if .Values.fluentbit.hostNetwork }}
   dnsPolicy: ClusterFirstWithHostNet
   {{- end }}
-  image: {{ .Values.fluentbit.image.repository }}:{{ .Values.fluentbit.image.tag }}
-  {{- with .Values.fluentbit.imagePullSecrets }}
+  image: {{ template "fluent-operator.image" (tuple .Values.fluentbit.image "") }}
```

Comment:
I don't think we want the fluentbit image tag to track the fluent-operator version, since these images are released on separate cadences and the versions will not align between fluent-operator and fluentbit.

Reply:
It doesn't, only the lines like this take the appVersion,

```yaml
image: {{ template "fluent-operator.image" (tuple .Values.operator.image $.Chart.AppVersion) }}
```

These other lines only take the value from values files and don't have a default (`""` means no default).

Using the helper macro, it will take the logic for combining the registry with the repository and such.